WeakAuras/WeakAuras.lua

```diff
@@ -4686,33 +4686,61 @@ local function actionGlowStart(actions, frame, id)
 end
 
 local glow_frame_monitor
+local anchor_unitframe_monitor
+WeakAuras.dyngroup_unitframe_monitor = {}
 do
   LGF.RegisterCallback("WeakAuras", "FRAME_GUID_UPDATE", function(event, frame, guid)
-    if type(glow_frame_monitor) ~= "table" then return end
-
-    local glows = glow_frame_monitor[guid]
-    if glows then
-      for id, data in pairs(glows) do
-        if data.frame ~= frame then
-          local new_frame = WeakAuras.GetUnitFrame(data.region.state.unit)
-          if new_frame and new_frame ~= data.frame then
-            -- remove previous glow
-            actionGlowStop(data.actions, data.frame, id)
-            -- apply the glow to new_frame
-            data.frame = new_frame
-            actionGlowStart(data.actions, data.frame, id)
-            -- update hidefunc
-            data.region.active_glows_hidefunc[data.frame] = function()
+    if type(glow_frame_monitor) == "table" then
```

Comment:
hmm, this callback gives us a update on a frame to guid mapping. Which is not exactly what we would need, if I see that correctly.

E.g. consider a aura with "unit == target", and attached to the target unit frame.
Now, the user changes the target, but the trigger says nothing has changed in the state.
The callback then might notify us that the old target's guid is now e.g. raid15's frame, but we don't want to move the aura, we want it to stay in the target frame.